standalone-metastore/metastore-server/src/main/sql/postgres/hive-schema-4.0.0.postgres.sql

```diff
@@ -1945,6 +1945,22 @@ CREATE TABLE "STORED_PROCS" (
 CREATE UNIQUE INDEX "UNIQUESTOREDPROC" ON "STORED_PROCS" ("NAME", "DB_ID");
 ALTER TABLE ONLY "STORED_PROCS" ADD CONSTRAINT "STOREDPROC_FK1" FOREIGN KEY ("DB_ID") REFERENCES "DBS" ("DB_ID") DEFERRABLE;
 
+-- Create stored procedure tables
+CREATE TABLE "PACKAGES" (
+  "PKG_ID" BIGINT NOT NULL,
+  "CREATE_TIME" INTEGER NOT NULL,
+  "DB_ID" BIGINT NOT NULL,
+  "NAME" VARCHAR(256) NOT NULL,
+  "OWNER_NAME" VARCHAR(128) NOT NULL,
+  "HEADER" TEXT NOT NULL,
+  "BODY" TEXT NOT NULL,
+  PRIMARY KEY ("PKG_ID")
+);
+
+CREATE UNIQUE INDEX "UNIQUEPKG" ON "PACKAGES" ("NAME", "DB_ID");
+ALTER TABLE ONLY "PACKAGES" ADD CONSTRAINT "PACKAGES_FK1" FOREIGN KEY ("DB_ID") REFERENCES "DBS" ("DB_ID")  DEFERRABLE;
```

Comment:
is there a reason to DEFER the FK instead of casecadeing the drop by using the FK?

Reply:
fixed